awx/main/migrations/0012_move_deprecated_stdout.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.7 on 2017-12-11 19:49
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0011_blank_start_args'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='DeprecatedJobStdout',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('result_stdout_text', models.TextField(blank=True, default=b'', editable=False)),
+            ],
+        ),
+        migrations.AddField(
+            model_name='deprecatedjobstdout',
+            name='job',
+            field=models.OneToOneField(editable=False, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='deprecated_result_stdout', to='main.UnifiedJob'),
+        ),
+        migrations.RunSQL('INSERT INTO main_deprecatedjobstdout(job_id, result_stdout_text) SELECT id, result_stdout_text FROM main_unifiedjob WHERE result_stdout_text IS NOT NULL;'),
```

Comment:
I've got a feeling @matburt ain't gonna like this :)

That said, I _really_ feel like we should avoid doing this in Python - it's just not going to be fast (or memory-performant) enough.

Reply:
I'll also add - I'm not sure how long it actually would take *postgres* to do this for large datasets - I plan to load a GB or so of stdout and attempt this query.